words

```diff
@@ -1,22 +1,22 @@
 Objective:
-	I’m trying to break out of that catch-22 of needing experience to gain experience.  With ambitions to eventually end up in data sciences, I understand that the path entails a humble beginning and a ton of studying.  I seek long-term mentorship for my skill development in a setting where I am able to grow and meaningfully contribute back.
+	With ambitions to excel in the field of data sciences, I understand that the path entails a humble beginning and a ton of studying.  I seek long-term mentorship for my skill development in a setting where I am able to grow and meaningfully contribute.
 
 Why me?
-	My previous career as a cook has planted me in fast-paced environments that necessitate adaptability; I am extremely comfortable wearing many hats and working with short and fluctuating deadlines.  My last few employers have placed me in management positions, and I relish in sharing  knowledge with my coworkers.  I would love to be a mentor in this field myself one day!
-	I am a highly motivated lifelong learner who is reliable, curious, creative, and honest.  I am by no means adverse to hard work and am currently both working and studying full time.  Driven to expand my abilities, all my personal projects have stretched me beyond the skills acquired at school.  I love tackling large problems and see a personal lack of skills as an opportunity to grow.
+	My previous career as a cook has planted me in fast-paced environments that necessitate adaptability; I am extremely comfortable wearing many hats and working with short and fluctuating deadlines.  My last few employers have placed me in management positions. I relish in sharing knowledge with others, and am looking forward to collaborating with and eventually mentoring other developers.
```

Comment:
I kinda feel like it might be strong to move the first sentence down. Opening with "I'm a cook" might not be the best. 

I changed the mentoring line because I felt "in this field" might have applied to your previous career. 

Also: talk about your code! Talk about how cleanly you code, and with lots of comments. Talk about how you find creative and efficient solutions, not just easy solutions. This sets you apart! It's really important. You care! Not many do.